src/test_utils.rs

```diff
@@ -39,6 +41,49 @@ pub const AVATAR_900x900_BYTES: &[u8] = include_bytes!("../test-data/image/avata
 static CONTEXT_NAMES: Lazy<std::sync::RwLock<BTreeMap<u32, String>>> =
     Lazy::new(|| std::sync::RwLock::new(BTreeMap::new()));
 
+pub struct AcManager {
```

Comment:
Minor nit, this is an `AcManager` but hands out `TestContext` instances.  That's not very consistent terminology...  Maybe `TestContextManager` would be more accurate?

Reply:
But `TestContextManager` is such a long word, reminds me of Java :joy: OTOH you're also right about terminology.

Reply:
Even worse, we already have something else that's also called "Account manager", which may lead to confusion. Still not giving up that I won't come up with a shorter name.

Reply:
I think your fear of long names is a bit exaggerated.  But you could go with `TestManager`, `ContextPool` (`TestContextPool`), or... naming is hard